browser/brave_content_browser_client.cc

```diff
@@ -846,6 +846,17 @@ bool BraveContentBrowserClient::HandleURLOverrideRewrite(
     return true;
   }
 
+#if !BUILDFLAG(IS_ANDROID)
+  if (url->host() == kAdblockHost) {
+    GURL::Replacements replacements;
+    replacements.SetSchemeStr(content::kChromeUIScheme);
+    replacements.SetHostStr(chrome::kChromeUISettingsHost);
+    replacements.SetPathStr(kContentFiltersPath);
+    *url = url->ReplaceComponents(replacements);
+    return false;
```

Comment:
returns false because we want to display original url and not the virtual in the location bar